tutorials/scripting/scene_unique_nodes.rst

```diff
@@ -0,0 +1,35 @@
+.. _doc_scene_unique_nodes:
+
+Scene Unique Nodes
+==================
+
+Introduction
+------------
+
+There are times in a project where a node needs to be called
+from a script, however it's position in the tree might change
+over time as adjustments are made to a scene, such as a
+button in a UI scene.
+
+In situations like this a node can be turned into a scene
+unique node to avoid updating a script multiple times.
```

Comment:
```suggestion
There are times in a project where a node needs to be called
from a script. However, its position in the tree might change
over time as adjustments are made to a scene, such as a
button in a UI scene.

In situations like this, a node can be turned into a scene
unique node to avoid updating a script multiple times.
```